packages/callout/src/Callout.tsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import Icon, { glyphs } from '@leafygreen-ui/icon';
+
+export const Variant = {
+  Note: 'note',
+  Tip: 'tip',
+  Important: 'important',
+  Warning: 'warning',
+} as const;
+
+export type Variant = typeof Variant[keyof typeof Variant];
+
+const baseStyle = css`
+  border-radius: 6px;
+  box-shadow: 0px 2px 5px 0 ${uiColors.gray.light2};
```

Comment:
I didn't catch this when reviewing the callout Sketch library, but we should be setting the color as a dark-gray or black RGBA value to work better on more backgrounds than white.